babel-watch.js

```diff
@@ -244,6 +245,11 @@ function restartApp() {
     }
   }
 
+  // If debugger mode is present 
+  if(program.debug) {
+      //Set an unused port number for child process.
+      process.execArgv.push('--debug=' + (program.debug));
```

Comment:
Isn't `restartApp()` called repeatedly? This doesn't seem idempotent. Would it be more appropriate to update the existing line to:

```
const app = fork(path.resolve(__dirname, 'runner.js'), {execArgv: process.execArgv.concat('--debug=' + program.debug) });
```

or something like that?